### PR TITLE
Added M79 prototype, and sound effect.

### LINKED ISSUE #879: M79

### PR DESCRIPTION
**Pull request by amo020**
Added the M79 prototype
Added the M79 sound effect imported from CM13

tweaked the spritwork are sling and backpack were inverted.

- fix #879

## Media

https://github.com/CM-14/CM-14/assets/167775671/3ee24268-f9a7-41a2-a132-ecdee43e9a19

**Changelog**

:cl:
- add: Added the M79 grenade launcher prototype

### CODE REVIEW

**Comment by amo020** [comment]
linked to #879

File: Resources/Audio/_CM14/Weapons/Guns/Cock/m79_break_open.ogg
Comment (QuietlyWhisper) [inline Resources/Audio/_CM14/Weapons/Guns/Cock/m79_break_open.ogg:1]: This file is missing an attributions.yml source

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Audio/_CM14/Weapons/Guns/Cock/m79_break_open.ogg
Comment (amo020) [reply Resources/Audio/_CM14/Weapons/Guns/Cock/m79_break_open.ogg:1]: a yes. and i did not used it also because there is no 'emptyGun' state

**Review by QuietlyWhisper — APPROVED** [review]
Looks good thanks

**Pull request merged**